solutions/devsprint-matheus-reis-2/DeliveryAppChallenge/Service/DeliveryApi.swift

```diff
@@ -33,8 +34,8 @@ struct DeliveryApi: DeliveryApiProtocol {
         
     }
     
-    func fetchMenuItem(_ completion: (String) -> Void) {
-        
+    func fetchMenuItem(_ name: String, _ completion: @escaping (Result<MenuItemDetails, DeliveryApiError>) -> Void) {
+        self.request(name, completion: completion)
```

Comment:
Como citado no outro comentário, removendo o parâmetro `name` do seu método, deve-se passar o `Endpoints.menuItemDetails`  para o método `request`.
```suggestion
        self.request(Endpoints.menuItemDetails, completion: completion)
```